.github/workflows/master_merge.yml

```diff
@@ -105,7 +105,7 @@ jobs:
       TAG: ${{needs.find-latest-tag.outputs.tag}}
 
   create-double-commit:
-    if: startsWith(github.event.pull_request.base.ref, 'rel')
+    if: startsWith(github.event.pull_request.base.ref, 'rel') && always()
```

Comment:
Wont `always()` run this step always (as name suggest). Even when branch does not start with `rel` ?

Reply:
No, only for rel base branches since it's in an `&&` conditional